docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '2.1'
```

Comment:
This feels a bit over complicated for a minimal example. We dont need the volume mounts, the numbers the end of `zoo1` and `kafka1` etc., the logger config etc. Feels to me that if this is here to provide a free template to test the interface, we shouldn't make it more 'production ready' if it makes it more complicated what is actually needed to run the image (given we both found that confusing when trying to stand this up).

Reply:
I should probably move this out of the repo, it's very much a convenience script.
I could move it to a folder which says so. :)